### DISCUSSION TITLE
New Payment Gateway for Duitku

### DISCUSSION THREAD

**Discussion by Akoswara97** [Paymenter/Paymenter#558] [Ideas]
Dear Dev,
i have suggestion about New Extension for Payment Gateway, Duitku is a populer payment gateway in Indonesia, Lot of Web Hoster and Server Hoster prefer use Duitku.
For Information of API Duitku : 
- https://docs.duitku.com/en/payment-gateway/api-browser/
- https://docs.duitku.com/api/en/#introduction

Thanks you

**Comment by 0xricoard** [comment]
Well i have created the Duitku Extension, i will publish it to Paymenter Marketplace soon. But you can check here https://github.com/0xricoard/Duitku-Paymenter

**Reply by MasGarX** [reply]
Mas bisa bikinin versi Midtrans?
